packages/web-components/fast-components/src/breadcrumb/breadcrumb.stories.ts

```diff
@@ -1,7 +1,34 @@
+import { STORY_RENDERED } from "@storybook/core-events";
+import addons from "@storybook/addons";
 import BreadcrumbTemplate from "./fixtures/base.html";
 import "../breadcrumb-item";
 import "./index";
 
+function addItem(): void {
+    const breadcrumbElement = document.getElementById("mybreadcrumb");
+    const items = breadcrumbElement?.querySelectorAll("fast-breadcrumb-item");
+
+    if (items === undefined) {
+        return;
+    }
+    const lastItem = items[items.length - 1];
+    const item: any = document.createElement("fast-breadcrumb-item");
+    item.setAttribute("href", "#");
+    item.textContent = `Breadcrumb item ${items.length + 1}`;
+
+    if (lastItem.parentNode === null) {
```

Comment:
I know this is just a helper function for your story but why not just say if (lastItem.parentNode !== null) then do the thing rather than have multiple return statements. I guess this is a preference thing so disregard if you don't agree

Reply:
thanks. refactored